token/program2/src/instruction.rs

```diff
@@ -187,6 +187,32 @@ pub enum TokenInstruction {
     ///   2. `[]` The account's multisignature owner.
     ///   3. ..3+M '[signer]' M signer accounts.
     CloseAccount,
+
```

Comment:
Is this at the end in an attempt to be backward compatible with spl-token 1.0?

Reply:
It means I don't need to make a million changes so TokenInstruction::unpack/pack